mxcube3-server

```diff
@@ -5,4 +5,4 @@ monkey.patch_all()
 from mxcube3 import app, socketio
 
 if __name__ == '__main__':
-    socketio.run(app, host='localhost', port=8081)
+    socketio.run(app, host='192.168.56.101', port=8081)
```

Comment:
I think we cannot let an IP address like this, in fact yesterday I changed 'localhost' to '0.0.0.0' in order to
bind to all interfaces, and PR was accepted. Would it be ok for you too ?
I would recommend to merge your repository before redoing the PR in order to sort out this kind of details.